tools/UpdateModules.ps1

```diff
@@ -44,13 +52,6 @@ function Create-ModulePsm1
   }
 }
 
-param(
```

Comment:
param needs to be at the beginning  to e recongnized as params atleast on my box :). leaving it as such